src/components/Checklist/md-in-js/fieldwork-001.md.ts

```diff
@@ -0,0 +1,26 @@
+export const title = `Basic recommendations for fieldwork`;
+
+// language=Markdown
+export const info = ``;
+// Mark nested list item with 'NESTEDLISTITEM'
+const rawMarkdownContent = `
+* Bring sunscreen, hat, sunglasses, bug spray  
+* Tell at least one person where you are going, when you are going, other information about your sampling trip (designate a rendezvous point or time)  
+* Know local wildlife, dangers they may pose and consider bringing wildlife deterrents (e.g., bear spray)  
```

Comment:
I found the original phrasing to be confusing.

```suggestion
* Know local wildlife and the dangers they may pose, and consider bringing wildlife deterrents (e.g., bear spray)  
```

Reply:
https://docs.google.com/document/d/1EqQ5Jw19qhe8o2LXUOaVWPxOrjM0mPzvFqE5WIZN7sw/edit